package.json

```diff
@@ -51,7 +51,7 @@
     }
   },
   "scripts": {
-    "start": "./cli/edgemicro start",
+    "start": "node app.js",
```

Comment:
This change could break scripts that others use to manage edgemicro. This may be a good candidate for a separate module that depends on edgemicro.